docs/fastddscli/cli/cli.rst

```diff
@@ -0,0 +1,139 @@
+.. include:: ../../03-exports/roles.include
+
+.. _ffastddscli_cli:
+
+CLI
+===
+
+The *Fast DDS* command line interface provides a set commands and sub-commands to perform, *Fast DDS*
+related, maintenance and configuration tasks.
+
+An executable file for Linux and Windows that runs the *Fast DDS CLI* application is
+available in the ``tools`` folder.
+If the ``tools/fastdds`` folder path is added to the ``PATH``, or by sourcing the `<path/to/fastdds>/install/setup.bash`
+environment variable, *Fast DDS CLI* can be executed running the following commands:
+
+- Linux:
+
+    .. code-block:: bash
+
+        $ fastdds <command> [<command-args>]
+
+-  Windows:
+
+    .. code-block:: bash
+
+        > fastdds.bat <command> [<command-args>]
+
```

Comment:
Introduction to the table. "There are two verbs which functionality is outlined in the following table...."